group-coordinator/src/test/java/org/apache/kafka/coordinator/group/GroupMetadataManagerTest.java

```diff
@@ -1134,6 +1140,19 @@ public CoordinatorResult<LeaveGroupResponseData, Record> sendGenericGroupLeave(
             return groupMetadataManager.genericGroupLeave(context, request);
         }
 
+        private void testDescribeGroupsUncommittedGroup(String groupId) {
```

Comment:
nit: `verifyDescribeGroupsReturnsDeadGroup(...)`? We use verify for similar methods in the context and this method is not aware of the "uncommitted" so it should not be in the name in my opinion.